code/game/objects/items/devices/tvcamera.dm

```diff
@@ -0,0 +1,147 @@
+/obj/item/device/tvcamera
+	name = "press camera drone"
+	desc = "An Ingi Usang Entertainment Co. livestreaming press camera drone. Weapon of choice for war correspondents and reality show cameramen. It does not appear to have any internal memory storage."
+	icon_state = "camcorder"
+	item_state = "camcorder"
+	w_class = ITEMSIZE_LARGE
+	slot_flags = SLOT_BELT
+	var/channel = "General News Feed"
+	var/obj/machinery/camera/network/news/camera
+	var/obj/item/device/radio/radio
+
+/obj/item/device/tvcamera/Destroy()
+	GLOB.listening_objects -= src
+	QDEL_NULL(camera)
+	QDEL_NULL(radio)
+	. = ..()
+
+/obj/item/device/tvcamera/Initialize()
+	camera = new(src)
+	camera.c_tag = channel
+	camera.status = FALSE
+	radio = new(src)
+	radio.get_frequency(FALSE)
+	radio.set_frequency(ENT_FREQ)
+	GLOB.listening_objects += src
+	. = ..()
+
+/obj/item/device/tvcamera/examine(mob/user)
+	. = ..()
+	to_chat(user, "Video feed is currently: [camera.status ? "Online" : "Offline"]")
+	to_chat(user, "Audio feed is currently: [radio.get_broadcasting() ? "Online" : "Offline"]")
+
+/obj/item/device/tvcamera/attack_self(mob/user)
+	add_fingerprint(user)
+	user.set_machine(src)
+	var/dat = list()
+	dat += "Channel name is: <a href='?src=\ref[src];channel=1'>[channel ? channel : "unidentified broadcast"]</a><br>"
+	dat += "Video streaming is: <a href='?src=\ref[src];video=1'>[camera.status ? "Online" : "Offline"]</a><br>"
+	dat += "Microphone is: <a href='?src=\ref[src];sound=1'>[radio.get_broadcasting() ? "Online" : "Offline"]</a><br>"
+	dat += "Sound is being broadcasted on frequency: [format_frequency(radio.get_frequency())] ([get_frequency_name(radio.get_frequency())])<br>"
+	var/datum/browser/popup = new(user, "Press Camera Drone", "EyeBuddy", 300, 390, src)
+	popup.set_content(jointext(dat,null))
+	popup.open()
+
+/obj/item/device/tvcamera/Topic(bred, href_list, state = physical_state)
+	if(..())
+		return 1
+	if(href_list["channel"])
+		var/nc = sanitize(input(usr, "Channel name", "Select new channel name", channel) as text|null)
+		if(nc)
+			channel = nc
+			camera.c_tag = channel
+			to_chat(usr, "<span class='notice'>New channel name: '[channel]' has been set.</span>")
+	if(href_list["video"])
+		camera.set_status(!camera.status)
+		if(camera.status)
+			to_chat(usr,"<span class='notice'>Video streaming: Activated. Broadcasting on channel: '[channel]'</span>")
```

Comment:
You could also make these chat alerts into balloon alerts if you wanna.

Maybe also give them a sound so the user knows something is happening.